<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PairingActivity</class>
 <widget class="QWidget" name="PairingActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>944</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PairingActivity</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelBodyPair">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;Pair with [DEST]s&lt;/span&gt;&lt;/p&gt;&lt;p&gt;To pair this &lt;span style=&quot; font-weight:600;&quot;&gt;[SOURCE]&lt;/span&gt; with an &lt;span style=&quot; font-weight:600;&quot;&gt;[DEST]&lt;/span&gt;, please make sure the [DEST] is in pairing mode too.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PortableIDWidget" name="widgetMyCertificate" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ConnectionStatusWidget" name="widgetConnectionStatus" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBarSearching">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PingWidget" name="widgetPings" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PairingListWidget" name="widgetPairingList" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDone">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PortableIDWidget</class>
   <extends>QWidget</extends>
   <header location="global">node/PortableIDWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PairingListWidget</class>
   <extends>QWidget</extends>
   <header location="global">pairing/list/PairingListWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConnectionStatusWidget</class>
   <extends>QWidget</extends>
   <header location="global">connection/ConnectionStatusWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PingWidget</class>
   <extends>QWidget</extends>
   <header location="global">discovery/ping/PingWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonDone</sender>
   <signal>clicked()</signal>
   <receiver>PairingActivity</receiver>
   <slot>complete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRefresh</sender>
   <signal>clicked()</signal>
   <receiver>PairingActivity</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>maybeOnward()</slot>
  <slot>tryAgain()</slot>
  <slot>complete()</slot>
  <slot>startCameraPair()</slot>
  <slot>refresh()</slot>
  <slot>toggleNetworkPair(bool)</slot>
  <slot>toggleBluetoothPair(bool)</slot>
  <slot>toggleAudioPair(bool)</slot>
 </slots>
</ui>
